src/Chapter2.hs

```diff
@@ -336,7 +347,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end list = if start < 0 then [] else take len (drop start list)
+    where
+      len = end - start + 1
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check :slightly_smiling_face: